main/ZgatewayBT.ino

```diff
@@ -1275,6 +1290,26 @@ JsonObject& process_inode_em(JsonObject& BLEdata) {
   return BLEdata;
 }
 
+JsonObject& process_ruuvi(JsonObject& BLEdata) {
+  const char* manufacturerdata = BLEdata["manufacturerdata"].as<const char*>();
+
+  double version = value_from_service_data(manufacturerdata, 5, 1);
+  double temperature = value_from_service_data(manufacturerdata, 6, 4) / 200;
+  double humidity = roundf((value_from_service_data(manufacturerdata, 10, 4) / 400)*100)/100;
+  double pressure = roundf(((value_from_service_data(manufacturerdata, 14, 4) + 50000) / 100)*100)/100;
+  double battery = value_from_service_data(manufacturerdata, 30, 2) + 1600;
```

Comment:
There was changes done in development branch related to `value_from_service_data` function. Could you cast it with `double` please?
You will need to rebase your branch with the current `development` one.
By doing that you will have also to add a `true` as the last argument of the `value_from_service_data` function. This will say to the function that the hex data needs to be reversed.